src/cryptoadvance/specter/server_endpoints/wallets/wallets_api.py

```diff
@@ -527,8 +527,9 @@ def addressinfo(wallet_alias):
             match = re.search(derivation_path_pattern, descriptor)
             if not match:
                 logger.debug(
-                    f"Derivation path of this descriptor {descriptor} could not be parsed. Sth. wrong with the regex pattern which was {derivation_path_pattern}?"
+                    f"Derivation path of this descriptor {descriptor} for address {address} could not be parsed. Sth. wrong with the regex pattern which was {derivation_path_pattern}?"
                 )
+                return jsonify(success=False)
```

Comment:
The only thing which can't be returned in this case could be the derivation path, would be overkill to only return False because of this IMO. The regex isn't perfect, too, for a multisig it is not capturing the derivation path correctly for example.

A partial return with True would be better, sth. like:
```

            return {
                "success": True,
                "address": address,
                "descriptor": descriptor,
                "xpubs_descriptor": xpubs_descriptor,
                "derivation_path": None,
                "walletName": wallet.name,
                "isMine": address_info and not address_info.is_external,
                **address_info,  # address_info is an instance of Address(dict)
            }
```

The derivation path is also only needed for the QR code message signing in Specter AFAIK.

Reply:
In https://github.com/cryptoadvance/specter-desktop/issues/2000
- descriptor == ""
- xpubs_descriptor == ""
- address_info == None
- wallet.name is misleading, because the address actually does NOT belong to the wallet

So actually the return is not really a success and does not contain useful information.